Question9.java

```diff
@@ -0,0 +1,20 @@
+class Solution {
```

Comment:
For each code, please add comments at the top. I am looking for feedbacks such as:

* how much time did you work on this problme
* how is the test results
* your difficulty score (from 1-10)
* other feedbacks such as which part you think you can do better or need do better

Reply:
Okay